src/explorer/connectionTreeItem.ts

```diff
@@ -106,44 +80,18 @@ export default class ConnectionTreeItem extends vscode.TreeItem
     return element;
   }
 
-  async listDatabasesUserHasAccessTo(
```

Comment:
Cool that we're sharing this with compass now 👍

Reply:
Yes! I was very excited to see this change in compass and know the fact that we can reuse it in vscode!